projects/src/transmitter.cc

```diff
@@ -0,0 +1,157 @@
+#include "transmitter.h"
+#include "hal.h"
+#include "packet/serialize.h"
+#include "usbconfig.h"
+
+// This define can be useful when sizing message mailbox and memory pools
+#define ASSERT_MESSAGE_MEMORY_LIMIT FALSE
```

Comment:
Eh, I suck at naming. I doesn't matter too much so I'll merge this PR but feel free to submit a name change PR after.